pkgs/servers/komga/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, fetchurl
+, lib
+, openjdk11
+}:
+
+let
+
+  pname = "komga";
+  version = "0.157.0";
+  jar = fetchurl {
+    url = "https://github.com/gotson/${pname}/releases/download/v${version}/${pname}-${version}.jar";
+    sha256 = "sha256-PkQL61fKplt6h1jcFCIMER+ZfzowDP466dR1AaDHw5Q=";
+  };
+
+in stdenv.mkDerivation rec {
+  inherit pname version;
+
+  phases = [ "installPhase" ];
+
+  installPhase = ''
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/lib/komga/"
+    cp -a ${jar} "$out/share/lib/komga/komga.jar"
+
+    cat <<EOF > "$out/bin/komga"
+    #!/bin/sh
+    ${openjdk11}/bin/java -jar "$out/share/lib/komga/komga.jar"
+    EOF
+    chmod +x "$out/bin/komga"
```

Prefer `makeWrapper` and `jdk11_headless` here.

```suggestion
    makeWrapper ${jdk11_headless}/bin/java $out/bin/komga --add-flags "-jar $out/share/lib/komga.jar"
```